recipe.py

```diff
@@ -17,11 +17,22 @@ def getDescription(self):
     def setDescription(self,description):
         self.description = description
 
-# r1 = Recipe("Lasagne")
-# r1.setDescription("Layers of pasta cheese and tomato sauce")
-# print(r1.getDescription())
+    def getNoServings(self):
+        return self.noServings
+    
+    def setNoServings(self, noServings):
+        if int(noServings):
+            self.noServings = noServings
+            print("This is an integer")
+        else:
+            print("Please input an integer")
```

Comment:
Sometimes a response needs to be in the form of a form of a success or failure to whatever calls these functions, this can be either by "raising" an exception when a value is incorrect or in a simplified form a boolean. Its also possible that printing a message might not be appropriate in all cases and as such it might be left to whatever is calling the setter to decide what to do.

Reply:
Another option is to hard set the expected type in the setter (so you always require something that is a integer) and leave it up to whatever calls this function to catch any error generated by not passing an integer. The type can be set using "typing" and some information can be found at https://docs.python.org/3/library/typing.html.

Reply:
Okay I will note this for future. I will read the article and try to type set the inputs.

Reply:
Also have a look at what I have done in my pr.